### PR TITLE
Replaced content with a link to CODE_OF_CONDUCT in the policies repo

### PR DESCRIPTION
**Pull request by baimamboukar**
### This is the continuation of #414 
- after moving the code of conduct to policies repository
- I replace the code_of_conduct in the website repository with a link to the code_of_conduct found in the policies repository

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,101 +1 @@
...
-to respond. If the person who is harassing you is part of the response team,
-they will recuse themselves from handling your incident. If the complaint
-originates from a member of the response team, it will be handled by a different
-member of the response team. We will respect confidentiality requests for the
-purpose of protecting victims of abuse.
-
-### Attribution & Acknowledgements
-
-This code of conduct is based on the Open Code of Conduct from the TODOGroup.
+[The code of conduct is now available here](https://github.com/open-life-science/policies-procedures-and-docs/blob/c342530ffe943bcc1a84a1a8a8e3b5b3c0fa4547/CODE_OF_CONDUCT.md)
```
[the remark this answers is not in the document]
Comment (baimamboukar) [reply CODE_OF_CONDUCT.md:1]: Alright ☄️
It is clear now! 
I'm updating the pull request now 🦸

[the remark this answers is not in the document]
Comment (baimamboukar) [reply CODE_OF_CONDUCT.md]: > You maybe have a sentence like here: https://github.com/galaxyproject/training-material/blob/main/CODE_OF_CONDUCT.md

Done ✅🚀
I updated this pull request...
Thanks for reviewing @bebatut

**Comment by bebatut** [comment]
Can you merge your two Pull Requests on the Code of Conduct inside one of them? Then everything happen together. Thanks a lot

**Comment by baimamboukar** [comment]
> > Can you merge your two Pull Requests on the Code of Conduct inside one of them? Then everything happen together. Thanks a lot
> 
> Like combine the two pull requests ? Let me read about it first....

Hello @bebatut 
This is [What I found out](https://github.com/orgs/community/discussions/22827)

**Comment by bebatut** [comment]
What I mean is putting all your changes around Code of Conduct in one PR

**Comment by baimamboukar** [comment]
> What I mean is putting all your changes around Code of Conduct in one PR

Honestly, I have never done this....
I have already raised two pull requests..
It is possible to combine then into one ? How this can be done?

**Comment by bebatut** [comment]
Could you take one of the Pull Request and make the changes you did in the other one directly there?

**Pull request closed without merging**